api/v1beta1/solrcloud_types.go

```diff
@@ -1523,6 +1523,14 @@ type SolrTLSOptions struct {
 	// This option is typically used with `spec.updateStrategy.restartSchedule` to restart Solr pods before the mounted TLS cert expires.
 	// +optional
 	MountedTLSDir *MountedTLSDirectory `json:"mountedTLSDir,omitempty"`
+
+	// Path on the Solr image to your JVM's truststore to merge with an external truststore.
+	// If supplied, Solr will be configured to use the merged truststore.
+	// The truststore for the JVM in the default Solr image is: $JAVA_HOME/lib/security/cacerts
+	MergeJavaTruststore string `json:"mergeJavaTrustStore,omitempty"`
```

Comment:
Does this work with `mountedTLSDir`?

Reply:
Typically, `mountedTLSDir` will have a csi driver volume and corresponding mount on the mainContainer, which would get used by the merge `initContainer`, so the init container would get the truststore file. However, that might cause double creation of the cert for each pod, once for the `initContainer` and once for the main container, so this would likely put a lot of pressure on the Cert issuer. So probably safer to say this feature is not supported with `mountedTLSDir` option for now.

Reply:
No, nvm all that `volumesAndMounts` doesn't include vols & mounts for the `mountedTLSDir` option. So this option is not going to work with `mountedTLSDir`.

So maybe we just punt on this feature for 0.6 and solve it using an init-db script instead? There's already some code in place for mounting a script into the init-db.